eclipse-collections-code-generator/src/main/resources/impl/list/mutable/unmodifiablePrimitiveList.stg

```diff
@@ -91,6 +91,12 @@ public class Unmodifiable<name>List
         return this.getMutable<name>List().lastIndexOf(value);
     }
 
+    @Override
+    public MutableList\<<wrapperName>\> boxed()
+    {
+        return new BoxedMutable<name>List(this).asUnmodifiable();
```

Comment:
I don't think you need to call `asUnmodifiable()` here, since we're already in an unmodifiable wrapper.

Reply:
Modified. Thanks.